webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/smartling/quartz/SmartlingPullBatchFileJobInput.java

```diff
@@ -0,0 +1,94 @@
+package com.box.l10n.mojito.service.thirdparty.smartling.quartz;
+
+import com.box.l10n.mojito.entity.Repository;
+import com.box.l10n.mojito.service.thirdparty.ThirdPartyTMSSmartling;
+import com.box.l10n.mojito.service.thirdparty.smartling.SmartlingOptions;
+import com.box.l10n.mojito.smartling.SmartlingClient;
+import java.util.Map;
+
+public class SmartlingPullBatchFileJobInput {
+
+  Long batchNumber;
+  String pluralSeparator;
+  Repository repository;
+  String projectId;
+  SmartlingOptions options;
+  Map<String, String> localeMapping;
+  ThirdPartyTMSSmartling.Prefix filePrefix;
+
+  SmartlingClient smartlingClient;
```

Comment:
Similarly here, passing client instances does not seem well suited for "job input". I'm even surprised this is working.

Reply:
This is from an old commit, head commit uses primitive types as this did not work 😀